### ISSUE TITLE
Google IAM error on task loading when running in Google Compute Engine

### ISSUE THREAD

**Issue by herbertrlee** [heartexlabs/label-studio#616]
**Describe the bug**
When running label studio on Google Compute Engine, tasks cannot be loaded from Google Cloud Storage.  The following error is flashed:

```
Tasks loading error
Error calling the IAM signBytes API: b'{\n "error": {\n "code": 403,\n "message": "The caller does not have permission",\n "status": "PERMISSION_DENIED"\n }\n}\n'
```

On the server side, I get the following stacktrace:

[label_studio_stacktrace.txt](https://github.com/heartexlabs/label-studio/files/6036763/label_studio_stacktrace.txt)

This issue DOES NOT happen when running locally, either in Docker or with bare Python.

**To Reproduce**
1. Deploy Label Studio on Google Compute Engine.  Specify a service account key file using the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
2. Import your tasks from a Google Cloud Storage bucket.
3. Go to the 'Tasks' page.
4. See error.  Tasks do not load.

**Expected behavior**
On the 'Tasks' page, I expect to see my tasks pop up.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Ubuntu
 - Label Studio Docker image SHA: `7466e7c045ed9e4d5bcd7ff0e48359929084c1f348c9c6035ba60df54d4bb6ef`
 - Google Compute Engine

**Additional context**
From my own investigation into the code, the issue seems to be in the `label_studio.utils.uri_resolver` module.  There is a function in this module called `resolve_gs` that checks if label studio is running on a Compute Engine instance.  If it is, the instance's Google credentials are used, rather than the credentials specified by the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.

This is inconsistent with both Google and Label Studio documentation - the `GOOGLE_APPLICATION_CREDENTIALS` should always be respected over the Compute Engine instance credentials.

**Comment by mp-pinheiro** [comment]
This can probably be fixed by simply sending `None` instead of the default service account address as the email credentials when generating the signed URL.

**Original:**
https://github.com/heartexlabs/label-studio/blob/c7c0734470f9cf6e2103072d71a0c93936c8e986/label_studio/io_storages/gcs/models.py#L147-L149

**Requested change:**
```py
signing_credentials = compute_engine.IDTokenCredentials(auth_request, "", service_account_email=None)
signed_url = signed_blob_path.generate_signed_url(expires_at_ms, credentials=signing_credentials, version="v4")
```

**Comment by makseq** [comment]
@mp-pinheiro It would be great if you could create a PR with this fix!

**Comment by mp-pinheiro** [comment]
@makseq Alright, will do. An alternative would be to give the administrator the option to choose the service account email.

**Comment by niklub** [comment]
The fix for this issue has been included in `label-studio==1.0.2` release, @mp-pinheiro thanks for your contribution!
